### ISSUE TITLE
Cutting/slashing

### ISSUE THREAD

**Issue by Nozu9** [studio-minus/ppg-bugs-and-requests#1432]
I've been thinking of a feature that would make swords and daggers able to cut. Basically if a sword/dagger is going at a certain velocity and impacts a human it leaves a cut injury and if it's moving at a high velocity it may even cut of a limb.
This could add more realism to the game and improve the gameplay by a bit. Been playing the game for about 900 hours and in my opinion it is now the one of the most important features that it lacks.

**Comment by DindinYT37** [comment]
Similar to #808 and #168.
*I have about 1.7k hours btw*

**Comment by Drogo-man** [comment]
The axe sorta does this, and it's hella buggy. Ive seen mods that do this and they are EVEN MORE buggy because the limbs starts to freak out when something impales multiple limbs at once.

**Comment by Nozu9** [comment]
> Similar to #808, #672 and #168. _I have about 1.7k hours btw_

mb, wasn't aware that this has been suggested before

**Comment by DindinYT37** [comment]
> The axe sorta does this, and it's hella buggy. Ive seen mods that do this and they are EVEN MORE buggy because the limbs starts to freak out when something impales multiple limbs at once.

Yeah because it was made to only impale one thing at once, this doesn't mean it's impossible to make. Rather it means that zooi has to either entirely, or almost entirely remake the system (though I can't judge how much work this would be since I don't know the original system).
